test/parallel/test-strace-openat-openssl.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const common = require('../common');
+const { spawn, spawnSync } = require('node:child_process');
+const { createInterface } = require('node:readline');
+const assert = require('node:assert');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+if (!common.isLinux)
+  common.skip('linux only');
+if (spawnSync('strace').error !== undefined) {
+  common.skip('missing strace');
+}
+
+{
+  const allowedOpenCalls = new Set([
+    '/etc/ssl/openssl.cnf',
+  ]);
+  const strace = spawn('strace', [
+    '-f', '-ff',
+    '-e', 'trace=openat',
+    '-s', '128',
```

Comment:
I know we've had issues with people running the tests in directories with long paths, and since this is a test, verbose output shouldn't be a big issue, so maybe it would be good to bump this value a bit higher?

```suggestion
    '-s', '512',
```